bgpd/bgp_dump.c

```diff
@@ -313,24 +313,24 @@ bgp_dump_route_node_record(int afi, struct bgp_dest *dest,
 	struct stream *obuf;
 	size_t sizep;
 	size_t endp;
-	int addpath_encoded;
+	bool addpath_capable;
 	const struct prefix *p = bgp_dest_get_prefix(dest);
 
 	obuf = bgp_dump_obuf;
 	stream_reset(obuf);
 
-	addpath_encoded = bgp_addpath_encode_rx(path->peer, afi, SAFI_UNICAST);
+	addpath_capable = bgp_addpath_encode_rx(path->peer, afi, SAFI_UNICAST);
```

Comment:
The return type of this api is still int.

Reply:
Just realized it's bgp_addpath_encode_rx, not bgp_addpath_encode_tx :) Fixed.